### PR TITLE
Testing header

### PR DESCRIPTION
**Pull request by Shazeen15**

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -66,14 +66,14 @@ h6 {
 
   .card,
   .card-body {
-    border: none;
+    border-color: rgba(125, 124, 124, 0.392);
```
Comment (zan-rosenthal) [inline app/assets/stylesheets/application.scss]: Maybe put this color in a variable?

File: app/javascript/components/pages/YelpBusinesses.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ import {
 export default class YelpBusinesses extends Component {
   render() {
     const { businesses } = this.props;
+    console.log(businesses)
```
Comment (zan-rosenthal) [inline app/javascript/components/pages/YelpBusinesses.js]: cut this.

File: app/javascript/components/pages/__test__/Header.test.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    })
+
+    it('displays navitem when signed in', () => {
+        const header = shallow(<Header logged_in={ true}
+            new_user_route="new_user_route"
+            sign_in_route="sign_in_route"
+            sign_out_route="sign_out_route"/>);
+        const nav = header.find('NavItem');
+        expect(nav.length).toEqual(5);
+        let navLinks = ['Home', 'Your Trips', 'New Trip', 'About Us', 'Sign Out']
```
Comment (zan-rosenthal) [inline app/javascript/components/pages/__test__/Header.test.js]: Could you move this up above line 25 and add an extra space below it for readability? Sorry I didn't think of this while we working on it.

**Review by zan-rosenthal — CHANGES_REQUESTED** [review]

File: app/javascript/components/pages/YelpBusinesses.js (hunk 2 above)
Comment (minachuong) [reply app/javascript/components/pages/YelpBusinesses.js]: 💯

**Review by zan-rosenthal — APPROVED** [review]

**Pull request merged**